projects/transformers/experiments/finetuning.py

```diff
@@ -78,16 +129,18 @@
     # logging
     report_to="tensorboard",
     task_name="glue",
+    run_name="debug_finetuning_bert100k_ntasks",
     # task_name=None,
     # task_names=["cola", "stsb", "mnli"],
+    max_steps=300,
     override_finetuning_results=False,
     task_hyperparams=dict(
-        wnli=dict(num_runs=2, learning_rate=2e-4),
-        rte=dict(num_runs=0),
+        wnli=dict(num_runs=2, max_steps=20, learning_rate=2e-4),
+        rte=dict(num_runs=1),
         cola=dict(num_runs=2),
-        stsb=dict(num_runs=1),
+        stsb=dict(num_runs=1,
+                  metric_for_best_model="pearson"),
```

Comment:
Do we have to define for each task? Is there a default set somewhere, so if we just say use_best_model it will automatically pick up that metric?